proposals/simd/Overview.md

```diff
@@ -1 +1,127 @@
+WebAssembly. It is based on [previous work on SIMD.js in the Ecma TC39
+ECMAScript committee](https://github.com/tc39/ecmascript_simd) and the
+[portable SIMD specification](https://github.com/stoklund/portable-simd) that
+resulted.
+
+There are three parts to the proposal:
+
+1. [A specification of portable SIMD operations](portable-simd.md) that came
+   out of the SIMD.js work.
+2. [A table of proposed WebAssembly operations](webassembly-opcodes.md) with
+   links to the portable specification.
+3. This document which describes the mapping between WebAssembly and the
+   portable specification.
+
+Referencing a common specification of the portable SIMD semantics reduces the
+work required to support both SIMD.js and WebAssembly SIMD in the same
+implementation.
+
+# Mapping portable SIMD to WebAssembly
+
+The types and operations in the portable SIMD specification are relatively
+straightforward to map to WebAssembly. This section describes the details of
+the mapping.
+
+## New value types
+
+The following value types are added to the WebAssembly type system to support
+128-bit SIMD operations. Each new WebAssembly value type corresponds to the
+[portable SIMD type](portable-simd.md#simd-types) of the same name.
+
+* `v128`: A 128-bit SIMD vector.
+* `b8x16`: A vector of 16 boolean lanes.
+* `b16x8`: A vector of 8 boolean lanes.
+* `b32x4`: A vector of 4 boolean lanes.
+* `b64x2`: A vector of 2 boolean lanes.
+
+The 128 bits in a `v128` value are interpreted differently by different
+operations. They can represent vectors of integers or IEEE floating point
+numbers.
+
+The four boolean vector types do not have a prescribed representation in
+memory; they can't be loaded or stored. This allows implementations to choose
+the most efficient representation, whether as a predicate vector or some
+variant of bits in a vector register.
+
+## Scalar type mapping
+
+Some operations in the portable SIMD specification use scalar types that don't
+exist in WebAssembly. These types are mapped into WebAssembly as follows:
+
+* `i8` and `i16`: SIMD operations that take these types as an input are passed
+  a WebAssembly `i32` instead and use only the low bits, ignoring the high
+  bits. The `extractLane` operation can return these types; it is provided in
+  variants that either sign-extend or zero-extend to an `i32`.
+
+* `boolean`: SIMD operations with a boolean argument will accept a WebAssembly
+  `i32` instead and treat zero as false and non-zero values as true. SIMD
+  operations that return a boolean will return an `i32` with the value 0 or 1.
+
+* `LaneIdx2` through `LaneIdx32`: All lane indexes are encoded as `varuint7`
+  immediate operands. Dynamic lane indexes are not used anywhere. An
+  out-of-range lane index is a validation error.
+
+* `RoundingMode`: Rounding modes are encoded as `varuint7` immediate operands.
+  An out-of-range rounding mode is a validation error.
```

Comment:
I would suggest starting this PR without rounding at all, mimic scalar, and filing a separate issue proposing how to encode it for both SIMD and scalar.